src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -40,16 +41,29 @@ export default class GeneSetSelector extends QueryStoreComponent<GeneSetSelector
 
 	@computed get geneListOptions()
 	{
-		return [
-			{
-				label: 'User-defined List',
-				value: ''
-			},
-			...gene_lists.map(item => ({
-				label: `${item.id} (${item.genes.length} genes)`,
-				value: item.genes.join(' ')
-			}))
-		];
+	    if (AppConfig.querySetsOfGenes == null || AppConfig.querySetsOfGenes == undefined) {
```

Comment:
I think you can just say `if (AppConfig. querySetsOfGenes == null)`, it covers `undefined` too as long as you don't use `===`